.github/workflows/test-runner.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.7, 3.8, 3.9]
+        python-version: [3.7, 3.8, 3.9, 3.11]
```

Comment:
Why not 3.10?

Reply:
It was interpreting 3.10 as 3.1?

Reply:
I see that stactools puts the versions in quotes. I will try that.

Reply:
looks like it's working